File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height: ");         //h marks the height of pyramid
    }    
    while (h > 8 || h <= 0);

    for (int i = 1; i <= h; i++) 
    {
        //This loop is for space to cover 
        for (int j = 1; j <= h - i; ++j)       
        {   
            // h-i place in each line of pyramid
            printf(" "); 
        }
        for (int k = 0; k < i; k++)            //this loop is for #
        {
            printf("#");
        }
        printf("\n");
    } 
}

File: mario.py
from cs50 import get_int

def main():

    while True:
        height = get_int("Height: ")
        if height > 0 and height <= 8:
            break

    for i in range(height):
        numhash = i + 1
        numspace = height - numhash

        print(" " * numspace, end = "")
        print("#" * numhash, end = "")
        print(" ", end = "")
        print("#" * numhash)

if __name__ == "__main__":
    main()